### PR TITLE
[#166761565] Add services tab to navigation

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to add services tab to navigation section on bottom screen, isolating services screen from previous location.

Simulator Screen Shot - iPhone X - 2019-06-19 at 09 40 51

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#166761565](https://www.pivotaltracker.com/story/show/166761565): Come CIT voglio vedere come terzo elemento della tab navigation la sezione servizi

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/store/actions/navigation.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const navigateToCalendarPreferenceScreen = () =>
 
 export const navigateToServicesPreferencesScreen = () =>
```
Comment (fpersico) [inline ts/store/actions/navigation.ts]: Please also change the method name

File: ts/components/ui/Markdown/handlers/internalLink.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const IO_INTERNAL_LINK_PREFIX = "ioit://";
 const ALLOWED_ROUTE_NAMES: ReadonlyArray<string> = [
   ROUTES.MESSAGES_HOME,
   ROUTES.PREFERENCES_HOME,
-  ROUTES.PREFERENCES_SERVICES,
+  ROUTES.SERVICES_HOME,
```
Comment (fpersico) [inline ts/components/ui/Markdown/handlers/internalLink.ts]: With this change you are breaking old messages links to services screen. We need to still handle old routename but forward to the new route.

Comment (lmorelli986) [reply ts/components/ui/Markdown/handlers/internalLink.ts]: fixed in 13a1a6c2

File: ts/store/actions/navigation.ts (hunk 1 above)
Comment (lmorelli986) [reply ts/store/actions/navigation.ts]: fixed in 13a1a6c

**Pull request merged**